src/elements/wrongNetwork/WrongNetworkModal.tsx

```diff
@@ -0,0 +1,42 @@
+import { ModalV3 } from 'components/modal/ModalV3';
+import { ReactComponent as WarningIcon } from 'assets/icons/warning.svg';
+import { Button, ButtonVariant } from 'components/button/Button';
+import { ReactComponent as IconSearch } from 'assets/icons/search.svg';
+import { useEffect, useState } from 'react';
+import { Navigate } from 'components/navigate/Navigate';
+
+export const WrongNetworkModal = ({
+  unsupportedNetwork,
+}: {
+  unsupportedNetwork: boolean;
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  useEffect(() => {
```

Comment:
unsupportedNetwork is props - why do you need to express it in useEffect?

Reply:
You are correct I don't need to